crates/oxc_parser/src/js/class.rs

```diff
@@ -0,0 +1,462 @@
+use oxc_allocator::{Box, Vec};
```

Comment:
Impressive how you managed to write the parsing in less than 500 lines. The class parsing is like 2000 something lines in Rome!